dns-records/harbor-alpha-phac-gc-ca.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSRecordSet
+metadata:
+  name: harbor-alpha-phac-aspc-gc-ca
+  namespace: dns
+  annotations:
+    sourceCodeRepository: "https://github.com/simardeep1792/harbor-k8s"
+  labels:
+    controlled-by: "phac-dns"
+    project-name: "phx-simardeepsingh"
+    project-id: "phx-01had7ny8p"
+spec:
+  name: harbor.alpha.phac-aspc.gc.ca."
```

Comment:
Please change from `harbor.alpha.phac-aspc.gc.ca` to `alpha.phac-aspc.gc.ca`. Subdomain should not be included in this field.

Reply:
Hi @ericmalen ,

I made the changes however the checks failed. Also as per the instructions on the Github page, the zone name should be included in the DNS name:

_The DNS name field for the zone should have one of the previously mentioned subdomains as a prefix, the general convention is <zone-name>.<sub-domain-name>.. For instance, if my Zone name is example, then the DNS name could be example.alpha.phac-aspc.gc.ca._

Reply:
Thank you @ericmalen for the quick response.